server/client.go

```diff
@@ -390,6 +390,19 @@ func (c *client) initClient() {
 	}
 }
 
+// RemoteAddress expose the Address of the client connection,
+// nil when not connected or unknown
+func (c *client) RemoteAddress() net.Addr {
```

Comment:
For a client, we store host, as string c.host. That is what we use for monitoring and statsz. Not sure if its useful here or not. Looks like probably not.

Reply:
Yeah that was the first thing I wanted to do - but then figured I'll get much richer information for free by exposing the net.Addr stuff - c.host would need parsing and you wont know what it is - pipe, v6, v4 etc - net.Addr does include that